third_party/FMI/main.c

```diff
@@ -878,136 +893,128 @@ fmiInteger addFMURootFolderName(char* fmuOutputWorkingFolder,
 ///\param index Index of FMU
 ////////////////////////////////////////////////////////////////
 
-fmiInteger addLibPathCurrentWorkingFolder(
-	char* trimfmuOutputWorkingFolder_wLiB,
-	char* fmuWorkingFolder, fmiInteger *sizefmuWorkingFolder,
-	fmiInteger *index) {
-		char *trimfmuWorkingFolder;
-		char * librPath_w32;
-		char * librPath_w64;
-		struct stat st;
-		fmiInteger len_LibPath;
-		fmiBoolean bRes_w32;
-		fmiBoolean bRes_w64;
+fmiInteger
+addLibPathCurrentWorkingFolder(char *trimfmuOutputWorkingFolder_wLiB, char *fmuWorkingFolder, fmiInteger *sizefmuWorkingFolder, fmiInteger *index)
+{
+    char *trimfmuWorkingFolder;
+    char *librPath_w32;
+    char *librPath_w64;
+    struct stat st;
+    fmiInteger len_LibPath;
+    fmiBoolean bRes_w32;
+    fmiBoolean bRes_w64;
 #if __linux__
-		char * librPath_l32;
-		char * librPath_l64;
-		fmiBoolean bRes_l32;
-		fmiBoolean bRes_l64;
+    char *librPath_l32;
+    char *librPath_l64;
+    fmiBoolean bRes_l32;
+    fmiBoolean bRes_l64;
 #endif
 
-		// allocate memory for the FMU-working folder trimmed
-		trimfmuWorkingFolder = (char*)calloc(sizeof(char),*sizefmuWorkingFolder + 1);
+    // allocate memory for the FMU-working folder trimmed
+    trimfmuWorkingFolder = (char *)calloc(*sizefmuWorkingFolder + 1, sizeof(char));
```

Comment:
Reorder the arguments to match the calloc signature. num is passed first, size of element is second.